src/UploadedFileIntegrationTest.php

```diff
@@ -108,7 +108,7 @@ public function testGetSize()
 
         $file = $this->createSubject();
         $size = $file->getSize();
-        if ($size) {
+        if ($size || 0 === $size) {
```

Comment:
according to the doc, file size is `int|false` so we can check for explicit false:

```suggestion
        if (false !== $size) {
```